chapter-7/CrazyLibrariansPointers/CrazyLibrariansPointers/crlibpont.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+#include <iostream>
+#include <string>
+#include <locale>
+
+using namespace std;
+
+string askText(const char* prompt, istream& in = cin, ostream& out = cout);
+int askNumber(const char* prompt, istream& in = cin, ostream& out = cout);
+void tellStory(const string* name, string* noun, int number, string* bodyPart, string* verb);
```

Comment:
Мне потребовалось добавлять `const` для остальных аргументов:
```suggestion
void tellStory(const string* name, const string* noun, int number, const string* bodyPart, const string* verb);
```
У тебя не было проблемы со сборкой?